custom/icds_reports/tasks.py

```diff
@@ -663,6 +664,12 @@ def _agg_ls_vhnd_form(state_id, day):
         AggregateLsVhndForm.aggregate(state_id, force_to_date(day))
 
 
+@track_time
+def _agg_gov_vhnd_form(state_id, day):
```

Comment:
Could you also add this function to the pipe repository? Here is the example Pr: https://github.com/dimagi/pipes/pull/24.
you would have to first add this function in `custom/icds_reports/management/commands/run_aggregation_query.py`

Reply:
done